packages/hooks/use-prop/index.ts

```diff
@@ -3,5 +3,5 @@ import type { ComputedRef } from 'vue'
 
 export const useProp = <T>(name: string): ComputedRef<T | undefined> => {
   const vm = getCurrentInstance()!
-  return computed(() => (vm.proxy?.$props as any)[name] ?? undefined)
+  return computed(() => (vm?.proxy?.$props as any)[name] ?? undefined)
```

Comment:
The `?? undefined` seems unnecessary as well.

Reply:
ok, it's removed.